website/content/en/docs/golang/quickstart.md

```diff
@@ -704,7 +706,7 @@ func (r *ReconcileMemcached) Reconcile(request reconcile.Request) (reconcile.Res
 
 			// Remove memcachedFinalizer. Once all finalizers have been
 			// removed, the object will be deleted.
-			memcached.SetFinalizers(remove(memcached.GetFinalizers(), memcachedFinalizer))
+			controllerutil.RemoveFinalizer(memcached, memcachedFinalizer)
```

Comment:
Can you add an import for `controllerutil` so users know where to find this package

Reply:
Thanks, @estroz , added.